IBNet/ITWSEncoding.cs

```diff
@@ -41,12 +41,15 @@
 #endregion
 
 using System;
+using System.IO;
 
 namespace Daemaged.IBNet
 {
   public interface ITWSEncoding
-  {    
-    bool DecodeBool();    
+  {
+    void SetStream(Stream stream);
```

Comment:
I see you introduced this, why? What does this achieve?
I don't see you using this anywhere, so I need some sort of reasoning...

Reply:
In case that someone choose to create an encoder out of the TWSClient and to pass it to the TWSClient-c'tor (for registration to a log message event from the encoder), thus we have to set the TcpClient stream that created only after Connect.